src/Chainweb/Pact/Backend/Orphans.hs

```diff
@@ -46,137 +47,106 @@ import Pact.Types.Server
 -- GENERIC INSTANCES --
 -----------------------
 
-deriving instance Generic ModuleData
-
-deriving instance Generic (ModuleDef a)
-
-deriving instance Generic Ref
-
-deriving instance Generic RefStore
-
+deriving instance Generic (Governance g)
```

Comment:
I'd love to get rid of all of these. Orphans are an indication of a design flaw.

Reply:
If you talk to either @emilypi or @slpopejoy, they will tell you there are plans to move these instances back into Pact; a place I think everyone would agree they should be. Until then, expect more of these type of PRs. Sorry...

Reply:
Luckily it seems like a lot of the types did get `Generic` instances in recent Pact, so at least we were able to cut down on some boilerplate here.